containers-tests/tests/intmap-properties.hs

```diff
@@ -1596,22 +1619,3 @@ prop_traverseWithKey_degrade_to_mapWithKey fun mp =
   where f = applyFun2 fun
         g k v = Identity $ f k v
         Identity newMap = traverseWithKey g mp
-
-prop_traverseMaybeWithKey_identity :: IntMap A -> Property
```

Comment:
These need to be restored too.

Reply:
Why? They are testing functions that don't exist anymore and were only exposed through `Data.IntMap.Internal`.

Reply:
Do you not use them for `mergeA`? I strongly suspect we'll ultimately want to expose an indexed `wither`-style function sooner rather than later.

Reply:
No, I don't. Because the tree is nonuniform (one `IntMap_ L`, some `Node L`, some `Node R`), the `mergeA` tactics need to produce multiple mutually recursive functions that modify the different types of node and can't just reuse the functions from `Data.IntMap.Internal`, which only work on `IntMap`.

However, it would be pretty easy to define things in the reverse direction (`traverseMaybeWithKey f (IntMap m) = IntMap <$> missingAllL (traverseMaybeMissing f) m`), and given that I understand the point of these tests better now. I'll reinstate the tests testing `traverseMaybeMissing` directly, and possibly extend them to the other tactic functions. (On a side note, this makes me wonder whether there should be a public function `WhenMissing f a b -> IntMap a -> f (IntMap b)`.)